dsa-solutions/lc-solutions/3000-3099/3010-divide an array into subarrays with minimum cost 1.md

```diff
@@ -0,0 +1,161 @@
+---
+id: Divide an Array Into Subarrays With Minimum Cost I
```

Comment:
replace `id: Divide an Array Into Subarrays With Minimum Cost I` to `id: divide-an-array-into-subarrays-with-minimum-cost-I`